packages/calypso-color-schemes/README.md

```diff
@@ -1,6 +1,8 @@
 # Calypso Color Schemes
 
-This package contains a number of CSS custom properties and Sass variables used in Calypso.
+This package contains a number of CSS custom properties used in Calypso.
+
+![Color scheme thumbnails](screenshot@2x.png)
```

Comment:
The image is nice but consider it _will_ get outdated. Is that an issue?

Reply:
I don’t think it is. It represents the concept more than the actual themes and colors.